### PR TITLE
[cinder-csi-plugin]: add additional info for CSI doc

### PR DESCRIPTION
**Pull request by jichenjc**
**The binaries affected**:

IMPORTANT: Please also add the binary name in the title, e.g.
`[openstack-cloud-controller-manager]: Add UDP protocol support`
unless the PR affects multiple binaries.

- [ ] openstack-cloud-controller-manager
- [x] cinder-csi-plugin
- [ ] k8s-keystone-auth
- [ ] client-keystone-auth
- [ ] octavia-ingress-controller
- [ ] manila-csi-plugin
- [ ] manila-provisioner
- [ ] magnum-auto-healer
- [ ] barbican-kms-plugin

**What this PR does / why we need it**:

**Which issue this PR fixes**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/using-cinder-csi-plugin.md [hunk 1]
```diff
@@ -191,33 +191,56 @@ Following prerequisite needed for volume snapshot feature to work.
...
+For Snapshot Creation and Volume Restore, please follow below steps:
 
 * Create Storage Class, Snapshot Class and PVC    
 ```
 $ kubectl -f examples/cinder-csi-plugin/snapshot/example.yaml create
-```     
+storageclass.storage.k8s.io/csi-sc-cinderplugin created
+volumesnapshotclass.snapshot.storage.k8s.io/csi-cinder-snapclass created
+persistentvolumeclaim/pvc-snapshot-demo created
+``
```
Comment (ramineni) [inline docs/using-cinder-csi-plugin.md]: there seems to be issue in formatting of doc ., may be because of missing quote above

Comment (jichenjc) [reply docs/using-cinder-csi-plugin.md]: yes, will update, thanks

**Comment by ramineni** [comment]
Thanks.
/lgtm

**Pull request merged**